test/spec/modules/riseBidAdapter_spec.js

```diff
@@ -444,6 +446,16 @@ describe('riseAdapter', function () {
       expect(request.data.bids[0].sua).to.not.exist;
     });
 
+    it('should send right position of the bid', function() {
+      const bid = utils.deepClone(bidRequests[1]);
+      const mediaType = bid.mediaTypes && bid.mediaTypes.banner ? BANNER : VIDEO;
+      bid.mediaTypes[mediaType].pos = pos;
```

Comment:
Since this is done to create a new bid of your choice, and not for the production logic, there's no need for logic to test the bid's media type. You can simply add the property you want for testing, as you know the object:
`bid.mediaTypes.banner.pos = pos`

Reply:
done